phpunit-api-coverage.xml

```diff
@@ -13,4 +11,8 @@
             <directory suffix=".php">src</directory>
         </whitelist>
     </filter>
+
+    <php>
+        <const name="SDK_TESTING_ENV" value="api_coverage" />
```

Comment:
Probably just a question of style but did you consider using an env variable instead?